### PR TITLE
IRC Text colour themes update

### PR DESCRIPTION
**Pull request by AllanOcelot**
This PR removes the IRC-Text colours from all theme files and places them inside our base theme file instead. It also lowers the luminosity of some of the IRC text colours. 
image

### CODE REVIEW

**Comment by prawnsalad** [comment]
This is now changing the green colour for all themes when the issue was only on the default theme. Has it been checked if these new colours look OK across all themes now it's the same everywhere?

**Comment by prawnsalad** [comment]
Looks like there's a few colours that could do with being tweaked. I've created a quick HTML layout plugin for kiwi that shows headers/paragraphs/links/forms/text colours so it's easier to see everything on one page. It adds a 'UI' icon in the bottom right when you're on a channel. https://gist.github.com/prawnsalad/f995c6fe929b19cb19de94a7fc42f751

Dark theme:
image

Default theme:
image

**Comment by AllanOcelot** [comment]
Yeah I've basically made it so all themes use the same IRC colours from base, instead of declaring them all in each file -  I was just in the process of tweaking the dark theme colours when you posted the plugin, I'll load the plugin and retest the themes :)

**Comment by AllanOcelot** [comment]
I've tweaked colours for a few themes but most have a consistent colour selection : 

Light themes:
image
Dark themes:
image


Your plugin has been really useful for checking them, so cheers for that!

File: src/res/globalStyle.css [hunk 1]
```diff
@@ -138,8 +138,8 @@ select {
     height: 40px;
     padding: 3px 5px;
     box-sizing: border-box;
-    border-radius: 1px;
-    border-radius: none;
+    border-radius: 0;
```
Comment (prawnsalad) [inline src/res/globalStyle.css]: Isn't this the default already?

File: static/themes/common/base.css [hunk 2]
```diff
@@ -1053,3 +1057,23 @@
 .kiwi-bufferkey-button {
     border-color: var(--brand-default-bg);
 }
+
+
+/* IRC TEXT COLOURS */
+/* IRC Text Colours */
```
Comment (prawnsalad) [inline static/themes/common/base.css]: Duplicated titles

File: static/themes/dark/theme.css [hunk 3]
```diff
@@ -177,37 +177,26 @@ button.u-button.u-button-primary.u-submit.kiwi-welcome-znc-start,
...
-.irc-bg-colour-light-cyan { background-color: #6dffff; }
-.irc-bg-colour-light-blue { background-color: #59b4ff; }
-.irc-bg-colour-pink { background-color: #f0f; }
-.irc-bg-colour-grey { background-color: #a6a6a6; }
-.irc-bg-colour-light-grey { background-color: #e2e2e2; }
+.kiwi-controlinput-active-tool {
+    background-color: transparent;
+}
+
+/* IRS COLOURS */
```
Comment (prawnsalad) [inline static/themes/dark/theme.css]: *IRC

File: src/res/globalStyle.css [hunk 4]
```diff
@@ -149,7 +149,6 @@ select {
 
 .u-form .u-input,
 .u-form textarea {
-    border: 1px solid;
```
Comment (prawnsalad) [inline src/res/globalStyle.css]: Removing this border makes it very unclear where input boxes are now
image

**Comment by AllanOcelot** [comment]
Our new PR #969 is much better than this one.

**Pull request closed without merging**